util/tracing/Makefile

```
@@ -27,11 +29,12 @@ trace_to_influxdb: trace_to_influxdb.o trace_util.o
 	$(CC) -o trace_to_influxdb trace_to_influxdb.o trace_util.o $(LIBS)
 
 install: trace_to_csv trace_to_chrome trace_to_influxdb
-	mv trace_to_csv $(INSTALL_PREFIX)
-	mv trace_to_chrome $(INSTALL_PREFIX)
-	mv trace_to_influxdb $(INSTALL_PREFIX)
-	ln -f -s launch-fedsd.sh $(INSTALL_PREFIX)/fedsd
-	chmod +x launch-fedsd.sh
+	mv trace_to_csv $(BIN_INSTALL_PATH)
```

```suggestion
	cp trace_to_csv $(BIN_INSTALL_PATH)
```
https://stackoverflow.com/questions/39892692/how-to-implement-make-install-in-a-makefile